libs/geospatial/src/main/java/org/codice/ddf/libs/geo/util/GeospatialUtil.java

```diff
@@ -0,0 +1,160 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.libs.geo.util;
+
+import org.codice.ddf.libs.geo.GeoFormatException;
+
+public class GeospatialUtil {
+    /**
+     * Parses Latitude in the DMS format of DD:MM:SS.S N/S
+     *
+     * @param dmsLat Degrees Minutes Seconds formatted latitude.
+     * @return Latitude in decimal degrees
+     */
+    public static Double parseDMSLatitudeWithDecimalSeconds(String dmsLat)
+            throws GeoFormatException {
+        Double lat = null;
+
+        if (dmsLat != null) {
+            dmsLat = dmsLat.trim();
+            String hemi = dmsLat.substring(dmsLat.length() - 1);
+
+            if (!(hemi.equalsIgnoreCase("N") || hemi.equalsIgnoreCase("S"))) {
+                throw new GeoFormatException(
+                        "Unrecognized hemishere, " + hemi + ", should be 'N' or 'S'");
+            }
+
+            int hemisphereMult = 1;
+            if (hemi.equalsIgnoreCase("s")) {
+                hemisphereMult = -1;
+            }
+
+            String numberPortion = dmsLat.substring(0, dmsLat.length() - 1);
+            if (dmsLat.contains(":")) {
+                String[] dmsArr = numberPortion.split(":");
+
+                int degrees = 0;
+
+                try {
+                    degrees = Integer.parseInt(dmsArr[0]);
+                } catch (NumberFormatException nfe) {
+                    throw new GeoFormatException(
+                            "Unable to parse degrees: " + dmsArr[0] + " from: " + dmsLat, nfe);
```

Comment:
Since you're not actually logging anything here, this is fine.  You could opt for `String.format` if localization is a big concern.